src/app/shared/dashboard-list-item/dashboard-list-item.component.html

```diff
@@ -107,13 +105,7 @@
           <button *ngIf="canPay" (click)="openDialog('pay')" mat-menu-item>
             Pay
           </button>
-          <app-redeem-button
-            (startRedeem)="startRedeem()"
-            [loan]="loan"
-            *ngIf="canRedeem"
-            name="Withdraw Collateral"
-            className="dashboard"
-          >
+          <app-redeem-button [loan]="loan" *ngIf="canRedeem" name="Withdraw Collateral">
```

Comment:
@martinbobbio the flag canRedeem changes when startRedeem() is called, please add this